.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.1
```

Comment:
I think it's good to have `2.5.1` support but should we set this to the highest version that our apps use? Also, should we add to the list of built rubies on Travis? https://github.com/amaabca/rest-client-jogger/blob/master/.travis.yml#L3

Reply:
[list of current .ruby-versions](https://github.com/search?q=org%3Aamaabca+filename%3A.ruby-version&unscoped_q=filename%3A.ruby-version)

Reply:
I definitely agree with adding it to the travis build matrix. Maybe we shouldn't have a `.ruby-version` file in this gem as it run across many different Ruby interpreter versions.